oracle/src/on_new_data.rs

```diff
@@ -0,0 +1,7 @@
+pub trait OnNewData<Key: Clone, Value: Clone> {
```

Comment:
move this to https://github.com/laminar-protocol/open-runtime-module-library/blob/master/traits/src/lib.rs

Reply:
use `#[impl_trait_for_tuples::impl_for_tuples(30)]` to automatic tuple implementation

example https://github.com/paritytech/substrate/blob/cefe4dc2ee4a1ed27fb171bdf65aed81ba0da44a/srml/support/src/traits.rs#L66

Reply:
@xlc ok